.circleci/config.yml

```diff
@@ -118,7 +152,12 @@ jobs:
             set -eux
             ls -la /tmp/workspace
             pushd packaging/docker
-              cp /tmp/workspace/x86_64/*.rpm cloudify-manager-install.rpm
+              cp /tmp/workspace/x86_64/cloudify-manager-install*.rpm cloudify-manager-install.rpm
+              cp /tmp/workspace/x86_64/prometheus-*.rpm prometheus.rpm
+              cp /tmp/workspace/x86_64/blackbox_exporter-*.rpm blackbox_exporter.rpm
+              cp /tmp/workspace/x86_64/node_exporter-*.rpm node_exporter.rpm
+              cp /tmp/workspace/x86_64/postgres_exporter-*.rpm postgres_exporter.rpm
+              cp /tmp/workspace/x86_64/rabbitmq_exporter-*.rpm rabbitmq_exporter.rpm
```

Comment:
all of this shouldn't be required, there's no reason for them to be separate rpms, they should be part of the main rpm (as every other component is)

so, they should be copied into the main rpm, inside of the `build_rpm` job, and I think it will be really easy to do it: the `build_prometheus_rpms` job already persists to workspace, so all we need to change is to make the `build_rpm` job depend on (`requires`) the prometheus job

Reply:
oh, that's brilliant :+1: